db/main/migrate/20180329000000_create_github_installations.rb

```diff
@@ -0,0 +1,32 @@
+class CreateGithubInstallations < ActiveRecord::Migration
+  PSQL = ActiveRecord::Base.connection.select_value('SELECT version()')
+
+  def self.up
+    create_table :github_installations do |t|
+      t.belongs_to :owner, :polymorphic => true
+      t.string     :owner_type
+      t.integer    :owner_id
+      if /PostgreSQL 9.3/.match(PSQL)
+        t.json       :permissions
+      else
+        t.jsonb    :permissions
+      end
+      t.integer    :added_by
+      t.datetime   :deleted_on
```

Comment:
could we also add a `deleted_by` (or maybe we should rename this to `removed_on` and `removed_by`?)